ports/phnt/vcpkg.json

```
@@ -1,6 +1,7 @@
 {
   "name": "phnt",
-  "version-date": "2020-12-21",
+  "version": "1.1.0",
+  "version-date": "2024-05-22",
```

```suggestion
  "version": "1.1.0",
  "version-date": "2024-05-22",
```

Either `version` or `version-date` , not both at the same time.
Also: I didn't find the `1.0.0` version upstream.